src/frontend/js/data/CourseProductProvider/index.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import { renderHook } from '@testing-library/react';
+import { CourseProductProvider, useCourseProduct } from 'data/CourseProductProvider/index';
+import { noop } from 'utils';
+
+describe('useCourseProduct', () => {
+  beforeAll(() => {
+    jest.spyOn(console, 'error').mockImplementation(noop);
```

Comment:
Don't you think that this spy is only useful in the error testing test ? It may hide some errors in the first test which does not test throwings.

Reply:
Indeed!